pkg/kubelet/dockershim/docker_container_test.go

```diff
@@ -123,7 +123,7 @@ func TestContainerStatus(t *testing.T) {
 	}
 
 	// Create the container.
-	fClock.SetTime(time.Now())
+	fClock.SetTime(time.Now().Add(-1 * time.Hour))
```

Comment:
I just want to make sure `Create`, `Start` and `Finish` timestamps are different.
It seems that they may be the same, that's why the test could pass before.